ddmark/validation_teststruct.go

```diff
@@ -67,9 +68,20 @@ type ExclusiveFieldsTestStruct struct {
 	CField    int
 }
 
-// +ddmark:validation:LinkedFields={StrField,IntField}
-// +ddmark:validation:LinkedFields={PStrField,PIntField,AIntField}
-type LinkedFieldsTestStruct struct {
+// +ddmark:validation:LinkedFieldsValue={StrField=aaa,IntField}
+// +ddmark:validation:LinkedFieldsValue={PStrField,PIntField,AIntField}
+type LinkedFieldsValueTestStruct struct {
+	RandomIntField int // allows to actually check all-empty structs
```

Comment:
🤔 not sure I get it. If we remove this field what happens?

Reply:
This might seem a bit intricate; the struct-level marker is only run on a given struct if it has at lease 1 existing subfield. If no subfield is set, the struct is never unmarshalled and just skipped.
All considered this is an expected and desirable behavior, otherwise every possible marked struct subfields would become de-facto Required in the validation of a parent struct.
This field is not here to be tested, but to allow for all-empty struct testing (ie. all 5 testing fields are empty or nil) without seeing the struct be skipped in the integration test.